### ISSUE TITLE
PlaceHolders in Lore

### ISSUE THREAD

**Issue by MrHermis** [ToberoCat/GuiEngine#8]
Hi

For some reason whenever I place a placeholder inside of a lore, the whole lor disappears from the material/item.
Is there another way to place them or is it just missing?

**Comment by MrHermis** [comment]
Found a way so it worked.. Apparently I needed to add this **implicit-update="false"**

**Comment by ToberoCat** [comment]
Yeah, this is pretty weird. I'll have a look at it, as this isn't an intended behavior.

Thanks for reporting

**Comment by TubseyInnit** [comment]
Hiya, I'm having the same issue and I'm wondering where '**implicit-update="false"**' is placed?

```
<component type="item" y="3" x="3" name="§3COPPER MINE!" material="BARRIER" hidden="{@guiengine.coppermine}">
    	<lore>§7LOCKED</lore>
        <lore>§7{%player_name%}</lore>
</component>
```
The above gives the below:

![image](https://github.com/user-attachments/assets/4b67e8e6-8e5d-48dc-8116-3b3071a2b4a3)

```
<component type="item" y="3" x="3" name="§3COPPER MINE!" material="BARRIER" hidden="{@guiengine.coppermine}">
    	<lore>§7LOCKED</lore>
        <lore implicit-update="false">§7{%player_name%}</lore>
</component>
```

The above gives the below:

![image](https://github.com/user-attachments/assets/e2c92a9b-7d8c-41e3-b529-aa0ac13ebd19)

I know {%player_name%} works as I placed in in the name of the component name and successfully displayed the playername. Just wondering how you guys resolved it and if there is something I can do to fix it?

**Comment by ToberoCat** [comment]
The implicit update has to be played at the <gui> Tag.

Like <gui implicit-update=true>

**Comment by TubseyInnit** [comment]
Like inside the Lore Tag?

**Comment by ToberoCat** [comment]
Yeah, like you did it in the lore Tag - but just this time in the GUI tag at the top of the file.
The lore Tag itself doesn't have the implicit update property
